### PR TITLE
Spectrum logo

### PR DESCRIPTION
**Pull request by RodolfoBrian**
**Issue:**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3699,6 +3709,7 @@
             "title": "Zulip",
             "hex": "52C2AF",
             "source": "https://github.com/zulip/zulip/"
-        }
+        },
```
Comment (ericcornelissen) [inline _data/simple-icons.json]:
```suggestion
        }
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3699,6 +3709,7 @@
...
+
```
Comment (ericcornelissen) [inline _data/simple-icons.json]:
```suggestion
```

**Comment by ericcornelissen** [comment]
From [this comment](https://github.com/simple-icons/simple-icons/issues/1791#issuecomment-545482837):

> Please check the lastest issue. i am get struggling

Could you tell us what you're struggling with so we can help you @RodolfoBrian?

**Comment by RodolfoBrian** [comment]
> From [this comment](https://github.com/simple-icons/simple-icons/issues/1791#issuecomment-545482837):
> 
> > Please check the lastest issue. i am get struggling
> 
> Could you tell us what you're struggling with so we can help you @RodolfoBrian?

I followed all the requirements and i still don't get. ( I converted to monochromatic, the canvas is 24 x 24,  I optimised the icon with SVGO ) so  i don't know  what i am doing wrong?

**Comment by birjolaxew** [comment]
@RodolfoBrian Almost there! Only a few minor changes left before this is ready to merge:

- As @ericcornelissen pointed out, the JSON is currently malformed due to an extra comma at the end. You can go to [the files tab](https://github.com/simple-icons/simple-icons/pull/1792/files) and click "Add suggestion to batch" on each of his suggested changes, before finally clicking the "Commit suggestions (2)" that pops up - that way his suggestions will be applied without you having to mess with Git all over again.

- One of our requirements is that the logo takes up as much of the 24x24 viewbox as possible (and is centered, but the logo in this PR seems to be already) - could you resize the logo so that it has a height or a width of 24 (or in this case the logo looks square, so a height and a width of 24)?

- It looks like the brand writes their name as "Spectrum", while this PR currently has `name` set to "spectrum". Could you update it with the correct capitalization?

**Comment by RodolfoBrian** [comment]
> @RodolfoBrian Almost there! Only a few minor changes left before this is ready to merge:
> 
> * As @ericcornelissen pointed out, the JSON is currently malformed due to an extra comma at the end. You can go to [the files tab](https://github.com/simple-icons/simple-icons/pull/1792/files) and click "Add suggestion to batch" on each of his suggested changes, before finally clicking the "Commit suggestions (2)" that pops up - that way his suggestions will be applied without you having to mess with Git all over again.
> * One of our requirements is that the logo takes up as much of the 24x24 viewbox as possible - could you resize the logo so that it has a height or a width of 24 (or in this case the logo looks square, so a height and a width of 24)?
> * It looks like the brand writes their name as "Spectrum", while this PR currently has `name` set to "spectrum". Could you update it with the correct capitalization?
> * This PR currently has the brand color set to `#FFFFFF`. It looks like their brand uses a purple color as their primary identifying color instead - specifically `#7B16FF`, if their favicon is to be trusted. Could you update the `color` too?

Thank you for your explanation  is much clear now.

**Comment by ericcornelissen** [comment]
> Thank you for your explanation is much clear now.

Thank you for the updates @RodolfoBrian, good work :+1: Could you update the SVG so that the logo touches the edges of the viewbox (4 if the logo fits perfectly into a square without distorting it, and 2 otherwise) as well?

**Comment by RodolfoBrian** [comment]
> > Thank you for your explanation is much clear now.
> 
> Thank you for the updates @RodolfoBrian, good work 👍 Could you update the SVG so that the logo touches the edges of the viewbox (4 if the logo fits perfectly into a square without distorting it, and 2 otherwise) as well?

it passed all the tests, svg follows all the requirements  as in the [checklist](https://github.com/simple-icons/simple-icons/pull/1792#issue-329957366) above.

**Comment by ericcornelissen** [comment]
> it passed all the tests, svg follows all the requirements as in the checklist above.

I'm sorry @RodolfoBrian but those are not a complete set of requirements, we list more in [the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) and the last thing I asked you to change is listed in those guidelines and [was mentioned by @birjolaxew](https://github.com/simple-icons/simple-icons/pull/1792#issuecomment-545630873) as well.

As you can see below the SVG you contributed is too small for our purposes, we want it to touch at least two of the edges of the viewbox (either horizontally or vertically) or all four if the logo is square.

snap

Just because it passes all our tests does not mean your done. The automated tests cannot (as far as I know) check if the logo fits into the viewbox as we want it to... I hope you understand that we value your contribution, but we as maintainers get to decide when a Pull Request will be merged. We try to be as clear about our requirements up front, but it may not be perfect and that is why we may ask you to change things here :slightly_smiling_face:

**Comment by RodolfoBrian** [comment]
> > it passed all the tests, svg follows all the requirements as in the checklist above.
> 
> I'm sorry @RodolfoBrian but those are not a complete set of requirements, we list more in [the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) and the last thing I asked you to change is listed in those guidelines and [was mentioned by @birjolaxew](https://github.com/simple-icons/simple-icons/pull/1792#issuecomment-545630873) as well.
> 
> As you can see below the SVG you contributed is too small for our purposes, we want it to touch at least two of the edges of the viewbox (either horizontally or vertically) or all four if the logo is square.
> 
> snap
> 
> Just because it passes all our tests does not mean your done. The automated tests cannot (as far as I know) check if the logo fits into the viewbox as we want it to... I hope you understand that we value your contribution, but we as maintainers get to decide when a Pull Request will be merged. We try to be as clear about our requirements up front, but it may not be perfect and that is why we may ask you to change things here 🙂

Thank you for making me understand the process.

**Comment by birjolaxew** [comment]
:+1: Looks good! Thanks a lot for all your work on updating this @RodolfoBrian and @ericcornelissen!

**Pull request merged**